### PR TITLE
Adds more information to documentation pages.

### PR DESCRIPTION
**Pull request by Vagr9K**
Including:

* Finishes the docpage for `pathPrefix`
* Adds a note about `navigateTo`

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview failed.

Built with commit 48c493f9b64b5d327677407a2ae3b44ccea25a24

https://app.netlify.com/sites/gatsbygram/deploys/596164b5a700c4420928c61f

File: packages/gatsby-link/README.md [hunk 1]
```diff
@@ -38,3 +39,17 @@ render () {
   </div>
 }
 ```
+
+## Forcing internal redirect
```
Comment (KyleAMathews) [inline packages/gatsby-link/README.md]: The normal term for this is "Programmatic Navigation" — please change this header to that

e.g. https://stackoverflow.com/questions/31079081/programmatically-navigate-using-react-router

File: docs/docs/path-prefix.md [hunk 2]
```diff
@@ -36,3 +36,11 @@ module.exports = {
...
 }
 ```
+
+Then pass `--prefix-paths` cmd option to Gatsby.
+
+```sh
+gatsby build --prefix-paths
+```
+
+NOTE: When using only `build`, Gatsby ignores `pathPrefix`.
```
Comment (KyleAMathews) [inline docs/docs/path-prefix.md]: Perhaps re-word this to:

"NOTE: When running the command without the `--prefix-paths` flag, Gatsby ignores your `pathPrefix`."

**Comment by KyleAMathews** [comment]
Just pushed the changes myself :-)

File: packages/gatsby-link/README.md [hunk 3]
```diff
@@ -38,3 +39,17 @@ render () {
...
+
+## Programmatic navigation
+
+For cases when you can only use event handlers for navigation, you can use `navigateTo`:
+
+```jsx
+import { navigateTo } from "gatsby-link"
+
+render () {
+  <div onClick={navigateTo('/example')}>
```
Comment (KyleAMathews) [inline packages/gatsby-link/README.md]: oh, this code won't run actually though right?

Comment (Vagr9K) [inline packages/gatsby-link/README.md]: Yeah, it had to be something like [this](https://codepen.io/rah98/pen/MoPXjy).
I'll submit a small PR for that.

Comment (Vagr9K) [inline packages/gatsby-link/README.md]: https://github.com/gatsbyjs/gatsby/pull/1440

**Pull request merged**